bin/reth/src/prometheus_exporter.rs

```diff
@@ -0,0 +1,20 @@
+//! Prometheus exporter
+
+use eyre::WrapErr;
+use metrics_exporter_prometheus::PrometheusBuilder;
+use metrics_util::layers::{PrefixLayer, Stack};
+use std::net::SocketAddr;
+
+pub(crate) fn initialize(listen_addr: SocketAddr) -> eyre::Result<()> {
```

Comment:
eventually, the exporter will be spawned with our `TaskManager`.

I'd like a separate function that returns the recorder and exporter.

perhaps even a type that bundles those and allows to configure the Stack, like a builder or something

Reply:
Would it be ok to create a separate issue for this and tackle it in another PR?